frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/EditDate/EditDate.tsx

```diff
@@ -194,20 +194,24 @@ export const EditDate = ({ field }: EditDateProps): JSX.Element => {
           {getValues('dateValidation.selectedDateValidation') ===
           DateSelectedValidation.Custom ? (
             <>
-              <Controller
-                control={control}
-                name="dateValidation.customMinDate"
-                rules={customMinValidationOptions}
-                render={({ field }) => <DateInput {...field} />}
-              />
-              <Controller
-                control={control}
-                rules={{
-                  deps: ['dateValidation.customMinDate'],
-                }}
-                name="dateValidation.customMaxDate"
-                render={({ field }) => <DateInput {...field} />}
-              />
+              <Box gridColumn={{ base: undefined, md: '1/3', lg: 'unset' }}>
```

Comment:
nit: can remove `base` key if we only want responsiveness to begin at `md`
```suggestion
              <Box gridColumn={{ md: '1/3', lg: 'unset' }}>
```

Reply:
fixed, realised i forgot to check mobile widths as well